Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.9.0"
 Animations = "27a7e980-b3e6-11e9-2bcd-0b925532e340"
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
 FFMPEG = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Comment:
I think we can remove VideoIO here and have it as a test dependency only, right?

Reply:
yes that should work .